### ISSUE TITLE
bin/setup wirft ActiveRecord::StatementInvalid: PG::UndefinedColumn: ERROR:  column "increment_by" does not exist

### ISSUE THREAD

**Issue by ronald** [hacken-in/hacken-in#719]
Ausgangspunkt:
- checkout
- bin/setup

```
[...]
-- add_index("visits", ["user_id"], {:name=>"index_visits_on_user_id", :using=>:btree})
   -> 0.0086s
-- initialize_schema_migrations_table()
   -> 0.1976s
rake aborted!
ActiveRecord::StatementInvalid: PG::UndefinedColumn: ERROR:  column "increment_by" does not exist
LINE 1: ...ions_id_seq"', (SELECT COALESCE(MAX("id")+(SELECT increment_...
                                                             ^
:               SELECT setval('"public"."regions_id_seq"', (SELECT COALESCE(MAX("id")+(SELECT increment_by FROM "public"."regions_id_seq"), (SELECT min_value FROM "public"."regions_id_seq")) FROM "regions"), false)
/gems/gems/activerecord-4.2.8/lib/active_record/connection_adapters/postgresql_adapter.rb:592:in `async_exec'
/gems/gems/activerecord-4.2.8/lib/active_record/connection_adapters/postgresql_adapter.rb:592:in `block in exec_no_cache'
/gems/gems/activerecord-4.2.8/lib/active_record/connection_adapters/abstract_adapter.rb:484:in `block in log'
/gems/gems/activesupport-4.2.8/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
/gems/gems/activerecord-4.2.8/lib/active_record/connection_adapters/abstract_adapter.rb:478:in `log'
/gems/gems/activerecord-4.2.8/lib/active_record/connection_adapters/postgresql_adapter.rb:592:in `exec_no_cache'
/gems/gems/activerecord-4.2.8/lib/active_record/connection_adapters/postgresql_adapter.rb:584:in `execute_and_clear'
/gems/gems/activerecord-4.2.8/lib/active_record/connection_adapters/postgresql/database_statements.rb:50:in `select_value'
/gems/gems/activerecord-4.2.8/lib/active_record/connection_adapters/postgresql/schema_statements.rb:338:in `reset_pk_sequence!'
/app/db/seeds.rb:15:in `seed'
[...]
```

**Comment by moonglum** [comment]
Thanks for the report 😄 Will look into it 👍

**Comment by moonglum** [comment]
Fixed by b9969d49156a528e2f5fb48dd91d5d381293c8e9 😄 We need to update to Rails 5 at some point... But this will be messy thanks to ActiveAdmin.
